tests/annotations/tests.py

```diff
@@ -1016,6 +1016,13 @@ def test_annotation_and_alias_filter_related_in_subquery(self):
                 {"name": "Morgan Kaufmann"},
             ],
         )
+    
+    def test_annotation_exists_empty_query_result(self):
+        Author.objects.create(name='John Smith')
+        Company.objects.create(name="Django Software Foundation")
+        self.assertFalse(Author.objects
```

Comment:
Use `assertIs(False, ...)` instead to make sure the regression test is valid. The `unittest.TestCase.assertFalse` method is actually _is falsey_ and since `bool(None) is False` this doesn't constitute a proper regression test. In other words, the test will pass even without the `Exists.empty_result_set_value` change applied.